File: src/main/kotlin/com/stannis/parser/sln/SlnStructure.kt
package com.stannis.parser.sln

import com.stannis.dataModel.Statement

data class SlnStructure(
    override val `$type`: String? = "SlnStructure",
    var token: String,
    var name: String,
    var path: String,
    var alias: String,
    var antebuild: Map<String, String?>
): Statement

File: Interpreter/Interpreter/Models/SlnStructure.cs
using System.Collections.Generic;

namespace Interpreter.Models{
    public class SlnStructure: IStatement
    {
        public string token { set; get; }
        
        public string name { set; get; }
        
        public string path { set; get; }
        
        public string alias { set; get; }

        public Dictionary<string, string> antebuild = new Dictionary<string, string>();

    }
};